python/cudnn/gemm/cutedsl/dense/amax/api.py

```
@@ -95,21 +144,21 @@ def check_support(self) -> bool:
+        if sf_dtype is cutlass.Int8:
             self._logger.warning("Int8 sf_dtype will be interpreted as float8_e8m0fnu, not as native int8")
 
         self._value_error_if(
-            sf_dtype == torch.float8_e4m3fn and self.sf_vec_size == 32,
+            sf_dtype is cutlass.Float8E4M3FN and self.sf_vec_size == 32,
             "Unsupported sf_dtype and sf_vec_size combination: float8_e4m3fn and 32 is not supported",
         )
         self._value_error_if(
-            ab_dtype in {torch.float8_e5m2, torch.float8_e4m3fn} and self.sf_vec_size == 16,
+            ab_dtype in {cutlass.Float8E5M2, cutlass.Float8E4M3FN} and self.sf_vec_size == 16,
             f"Unsupported ab_dtype and sf_vec_size combination: {{float8_e5m2, float8_e4m3fn}} and 16 is not supported",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)